webrtc-stats.html

```diff
@@ -612,8 +612,54 @@ <h3>
           the first RTCP packet is sent or received that refers to the <a>SSRC</a> of the <a>RTP stream</a>.
         </p>
         <p>
-          RTP monitored objects are not [= deleted =].
+          RTP monitored objects are deleted when the RTP stream is reconfigured
+          such that the {{RTCRtpStreamStats/ssrc}} or
+          {{RTCRtpStreamStats/codecId}} changes or if the
+          {{RTCRtpTransceiver}}'s {{RTCRtpTransceiver/currentDirection}} becomes
+          <code>"stopped"</code>. The monitored object is <i>not</i> deleted if
+          the transceiver is made <code>"inactive"</code> or if the encoding
+          parameters' {{RTCRtpEncodingParameters/active}} attribute is set to
+          <code>false</code>. When an RTP stream is reconfigured such that the
+          monitored object is deleted but the RTP stream continues to be used, a
+          new RTP monitored object and {{RTCStats/id}} will be created for the
+          new SSRC or codec.
```

Comment:
TODO: Reconfiguring to send fewer simulcast layers should be treated as if the codec changed. We have to reset the first layer if the second layer disappears otherwise it can look like the number of packets sent decreased.

Reply:
I think I disagree but can explain a bit more please? Probably also explain whether you mean spatial or temporal layer ;-)

Reply:
I'm not talking about inactivating a layer, I'm talking about a renegotiation to permanently delete the layer (once it is gone, you cannot revive it). In the spirit of not having to implement caches for old SSRCs, we should delete that outbound-rtp. And if we do that, the only way to make sense of the remaining outbound-rtp layer is to delete and replace it, otherwise "sum of outbound-rtp packets with same mid before negotiation > sum of outbound-rtp packets with same mid after negotiation" which will trick the caller to calculate a negative packet rate. It's easier to think of this weird edge case as a codec reconfiguration.

Reply:
I'm talking about simulcast, not SVC. getStats does not support per-SVC layer stats yet, so it only has one outbound-rtp for everything and those counters would always go up as far as I'm concerned. To be revisited when we have per-SVC layer stats.